inception-project/inception
# #325 Recommendations for label-less annotations and feature-less layers

**What's in the PR**
* Support recommendations for annotations which do not already have a value(i.e. label) assigned.
* On a sidenote added a sentence on checkstyle troubleshooting to the development guide

**How to test manually**
* Configure a project with the Stringmatch recommender
* Create some annotations without a value/label

* Start an active learning session to verify that recommendations without a label/value are still considered

**Automatic testing**

**Documentation**
